index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lawffice</title>
+</head>
+<body>
+    <header><h1>Law Office of Maisie A. Barringer</h1></header>
+    <nav>
+        <ul>
+            <li>Client Reviews</li>
+            <li>Schedule a meeting</li>
+            <li>Contact Me</li>
+            <li>Informational Videos</li>
+            <li>Services</li>
+        </ul>
+    </nav>
+    <div id="reviews"></div>
+    <div id="schedule"></div>
+    <div id="contact"></div>
+    <div id="videos"></div>
+    <div id="services">
+        <ul>
```

Comment:
This maybe doesn't have to be a list unless we plan on doing something other than just displaying the services offered.

Reply:
Think it should all go in one p? I think having the services listed out vertically singles them out and makes it easier for the clients to read each service. I'd like to get a second opinion once another of our group reviews the code

Reply:
If we're basing the website on the Challenge #2 example website, I think listing them out vertically works well, but we should add a blurb/description underneath each of them. Honestly, while I like the idea of having all of this information on one page, I think it might end up looking really cluttered and busy if we don't give things like "reviews" and "services" their own separate page.